### PR TITLE
SW-2613 Disable outplant option on withdraw when no seeds are ready

### PR DESCRIPTION
**Pull request by constanzauanini**
When no seeds are ready, instead of selecting OUTPLANT purpose and show an error, disable that option and default selection to NURSERY_TRANSFER.

Error message is now shown in an info tooltip

Screenshot 2023-01-19 at 14 53 48

### CODE REVIEW

File: src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx [hunk 1]
```diff
@@ -512,12 +518,18 @@ export default function SelectPurposeForm(props: SelectPurposeFormProps): JSX.El
                   {strings.PURPOSE_REQUIRED}
                 </FormLabel>
                 <RadioGroup name='radio-buttons-purpose' value={localRecord.purpose} onChange={onChangePurpose}>
-                  {!contributor && <FormControlLabel value={OUTPLANT} control={<Radio />} label={strings.OUTPLANT} />}
+                  {!contributor && (
+                    <FormControlLabel
+                      value={OUTPLANT}
+                      control={<Radio />}
+                      label={getOutplantLabel()}
+                      disabled={noReadySeedlings}
```
Comment (karthikbtf) [inline src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx:526]: i think you may need to remove `saveDisabled` from line 472

Comment (constanzauanini) [reply src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx:524]: Oh, thank you!!

File: src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx [hunk 2]
```diff
@@ -413,15 +412,12 @@ export default function SelectPurposeForm(props: SelectPurposeFormProps): JSX.El
       if (!hasReadyQuantities) {
         if (!noReadySeedlings) {
           setNoReadySeedlings(true);
-          snackbar.toastError(
-            strings.NO_SEEDLINGS_AVAILABLE_TO_OUTPLANT_DESCRIPTION,
-            strings.NO_SEEDLINGS_AVAILABLE_TO_OUTPLANT_TITLE
-          );
+          updateField('purpose', NurseryWithdrawalPurposes.NURSERY_TRANSFER);
+          setIsNurseryTransfer(true);
```
Comment (karthikbtf) [inline src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx:416]: i think we are missing initialization of from nursery... can you maybe extract out the 'onChangePurpose' and reuse that function?

**Review by karthikbtf — APPROVED** [review]
tests ok now

**Pull request merged**